base/src/main/java/com/smartdevicelink/proxy/rpc/OnVehicleData.java

```diff
@@ -458,11 +467,29 @@ public Double getInstantFuelConsumption() {
         return SdlDataTypeConverter.objectToDouble(object);
     }
 
+    /**
+     * Sets the externalTemperature.
+     *
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
+     * @since SmartDeviceLink 2.0.0

```

Comment:
@kostyaBoss below line 474 you need this: `* {"num_min_value": -40.0, "num_max_value": 100.0}`